src/content/app/new-species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -28,22 +28,15 @@ import { PrimaryButton } from 'src/shared/components/button/Button';
 import styles from './SpeciesSearchField.scss';
 
 export type Props = {
+  query: string;
   onSearchSubmit: () => void;
   canSubmit?: boolean;
-  onSpeciesAdd?: () => void;
+  // onSpeciesAdd?: () => void;
   onInput?: ((event: FormEvent<HTMLInputElement>) => void) | (() => void);
 };
 
-const SpeciesSearchField = (props: Props) => {
-  const { canSubmit = true } = props;
-  const dispatch = useAppDispatch();
-  const query = useAppSelector(getSpeciesSearchQuery);
-
-  const onInput = (event: FormEvent<HTMLInputElement>) => {
-    const value = event.currentTarget.value;
-    dispatch(setQuery(value));
-    props.onInput?.(event);
-  };
+export const SpeciesSearchField = (props: Props) => {
```

Comment:
Won't it be a bit confusing when importing `SpeciesSearchField` component and the default component which could get named as `SpeciesSearchField` as well?

Reply:
It might be. I am not quite sure what the best approach would be. I did not want to leak the word "wrapped" outside of this module.

Reply:
I have not got a better name for `SpeciesSearchField` either unfortunately

Reply:
Let's leave it until it becomes a problem then 🙂